meteor-app/imports/startup/server/email.js

```diff
@@ -33,5 +37,96 @@ Meteor.methods({
     } catch (error) {
       log.error('Error from email gateway', error)
     }
+  },
+  sendMembershipEmail(to, name, type, expiry, apiKey) {
+    sgMail.setApiKey(Meteor.settings.private.sendgridApikey)
+    const options = {
+      to,
+      from: Meteor.settings.private.fromEmail,
+      templateId: apiKey,
+      dynamic_template_data: {
+        name,
+        type,
+        expiry
+      }
+    }
+    sgMail.send(options)
+  },
+
+  sendMembershipRenewals() {
+    Members.find({}).forEach(member => {
+      Purchases.find({
+        memberId: member._id,
+        code: '/PA-MEMB/',
+        expiry: { $lt: new Date() }
+      }).forEach(purchase => {
+        Meteor.call(
+          'sendMembershipEmail',
+          member.email,
+          member.name,
+          purchase.productName,
+          Moment(purchase.expiry).format('Do MMM YYYY'),
+          Meteor.settings.private.expiredMembershipID
+        )
+      })
+    })
+  },
+  sendMembershipReminderEmail() {
+    Members.find({}).forEach(member => {
+      Purchases.find({
+        memberId: member._id,
+        $or: [{ code: 'PA-MEMB-12' }, { code: 'PA-MEMB-3' }],
+        expiry: { $gt: new Date() }
+      }).forEach(purchase => {
+        Meteor.call(
+          'sendMembershipEmail',
+          member.email,
+          member.name,
+          purchase.productName,
+          Moment(purchase.expiry).format('Do MMM YYYY'),
+          Meteor.settings.private.validMembershipID
+        )
+        debug('Email sending to ' + purchase.purchaser)
+      })
+    })
+  },
+  sendPassEmail(to, name, visitsUsed, apiKey) {
+    sgMail.setApiKey(Meteor.settings.private.sendgridApikey)
+    const options = {
+      to,
+      from: Meteor.settings.private.fromEmail,
+      templateId: apiKey,
+      dynamic_template_data: {
+        name,
+        type,
+        expiry,
+        visitsUsed
+      }
+    }
+    sgMail.send(options)
+  },
+  sendPassRenewal() {
+    Members.find({}).forEach(member => {
+      if (member.sessions.length >= 5) {
+        Meteor.call(
+          'sendPassEmail',
+          member.email,
+          member.name,
+          member.sessions.length,
+          Meteor.settings.private.validPassID
+        )
+
+        debug('Sending Email to ' + member.name)
```

Comment:
It's worth being more descriptive of which email you are sending here, eg 'sending passes remaining email'...